packages/deprecated/stylus/package.js

```diff
@@ -0,0 +1,8 @@
+Package.describe({
+  summary: 'DEPRECATED - Expressive, dynamic, robust CSS',
+  version: "2.513.14"
```

Comment:
If we bump the minor to, say, `2.999.0`, it could prevent (some, post-1.4.x) users from automatically updating to the deprecated version (and instead, leave them on the prior 2.513.13) when running `meteor update --all-packages` (or `meteor update stylus`).  Then, since it wouldn't be "`core`" in future published Meteor versions, I believe they'd be allowed to keep using it `@2.513.13` after that.  Am I mistaken?

Otherwise LGTM.

Reply:
That makes sense to me @abernix - changes coming - thanks!